bundle/LegacyMapper/Configuration.php

```diff
@@ -180,6 +180,9 @@ public function onBuildKernel( PreBuildKernelEvent $event )
         // Multisite settings (PathPrefix and co)
         $settings += $this->getMultiSiteSettings();
 
+        // SiteAccess settings
+        $settings += $this->getSiteSettings();
```

Comment:
If there is multisite, yes it will use the same value again (this could be cleanup up) but if there is no multisite they won't be set yet.

I guess getMultiSiteSettings() should set legacy IndexPage and DefaultPage only if there is no custom setting, is that correct?